generators/client-2/templates/src/main/webapp/app/_app.state.ts

```diff
@@ -1,5 +1,8 @@
 import { NavbarComponent } from './layouts';
-import { AuthService, <%=jhiPrefixCapitalized%>LanguageService } from './shared';
+import { AuthService } from './shared';
+<%_ if (enableTranslation) { _%>
```

Comment:
@deepu105 What about the import statement. Where do you want to put that? Should this be part of client-2 so that it is applied to only Angular 2 or should we check if it is Angular2 code apply this regex otherwise previous one?